src/common/queryBuilder.js

```diff
@@ -45,28 +45,54 @@ function buildQuery({ analyzers, fields, operator }) {
          * @returns {object} The resulting query bool object for ElasticSearch.
          */
         queryFn: (query) => {
+            const inclusions = [];
+            const exclusions = [];
+
+            // separate query into inclusions and exclusions
+            // based on hyphen indicator
+            if (query.match(/-/g)) {
+                query.split(" ").forEach((q) => {
+                    if (q.match(/-/g)) {
+                        exclusions.push(q.slice(1));
+                    } else {
+                        inclusions.push(q);
+                    }
+                });
+            } else {
+                inclusions.push(query);
+            }
+
             // build array of match queries for each analyzer,
             // then flatten them into a single array of queries
-            const queries = analyzers
+            const includedQueries = analyzers
+                .map((analyzer) =>
+                    buildMatchQueries({
+                        analyzer,
+                        fields,
+                        operator: operator || "or",
+                        query: inclusions.join(""),
```

Comment:
```suggestion
                        query: inclusions.join(" "),
```

Needs to be joined with a space, otherwise terms will be mashed together (i.e. `samuel beckett -letter` would lead to `samuelbeckett` included and `letter` excluded)